howto/manage/setup-tls-for-appliance.md

```diff
@@ -0,0 +1,81 @@
+The Anbox Cloud Appliance uses the self-signed certificate to provide HTTPS services hence it is considered to be used for development or testing purposes in the majority of cases due to the security risks. In case you want to use it for production, you will need to generate a valid SSL certificate and install it to the Anbox Cloud Appliance.
```

Comment:
Using a signed certificate also makes sense for development and testing, it's not limited to just production use cases. I wouldn't limit the scope here at all but reword this so it describes more from a perspective how to add it when needed not telling when it is needed.